java/src/server/persistence/database/DatabaseFacade.java

```diff
@@ -93,6 +93,21 @@ public void loadJar(String url){
         }
     }
 
+    @Override
+    public void clear() {
+        //// TODO: 4/9/16 implement this
+    }
+
+    @Override
+    public void startTransaction() {
+        //// TODO: 4/9/16 don't know if we need this.
+    }
+
+    @Override
+    public void endTransaction(boolean commitTransaction) {
+        //// TODO: 4/9/16 don't know if we need this
+    }
+
     public List<GameDTO> getAllGames() throws InvocationTargetException, IllegalAccessException {
```

Comment:
Does this get overridden now? I saw you added the functions to the Facade, just making sure the functions in here now have the Override tag.